docs/manual_test_plan.md

```diff
@@ -18,3 +18,67 @@ requests are not turned on in production.
+|8|In the Purpose field, type "Training"|The form will show the appropriate Purpose|
+|9|In the Event Dates field, type "12/14/2026 - 12/18/2026"|The form will show the selected date range|
+|10|In the Travel Dates field, type "12/13/2026 - 12/19/2026"|The form will show the selected date range|
+|11|In the Notes to Approvers field, Type "This is a note"|The form will show the entered note|
+|12|In the Anticipated Expenses section, enter some expenses|The form will show a total for each line and an overall total|
+|13|Press the Submit Request button|A modal will appear indicating a possible date in the Event Name|
+|14|Press the Continue button|You see "Travel request was successfully created."|
+
+#### Editing a travel request
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Create a travel request as detailed above|You see "Travel request was successfully created."|
+|2|Press the Edit button at the bottom of the page|You should be taken to a page where the fields are editable|
+|3|Make a few changes|The fields should reflect the new values|
+|4|Add a new note to the Approvers|The fom will show the entered note|
+|5|Add or remove some expenses|The total should update to reflect changes|
+|6|Press the Submit Request button|A modal will appear indicating a possible date in the Event Name if you did not remove the numbers|
+|7|Press the Continue button|You see "Travel request was successfully updated."|
+
+#### Setting up a delegation for testing
+|Step|Action|Expected result|
+|---|---|---|
+|1|Create a travel request as detailed above|You see "Travel request was successfully created."|
+|2|Follow [the instructions](https://github.com/pulibrary/approvals/tree/main?tab=readme-ov-file#testing-user-accounts) to make yourself a delegate for your approver|The rake task should be successful
+|3|Click on you netid at the top right to open the menu and select "My Delegations"|You should see your approver in the list of names|
+|4|Click on your approver's name to act as a delegate for them|You should see a banner at the top of the page saying you are acting as a delegate for the selected person|
+
+
+#### Approving a travel request
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|With the application set to act as a delegate for your approver as detailed above|You should see a banner at the top of the page saying you are acting as a delegate for the selected person|
+|2|Click on the "Requests to Review" link in the header|You should be taken to the "Requests Awaiting My Approval" page with the request you created|
+|3|Click on the title of the request you just created|You should navigate to a page with the details of the request|
+|4|Enter a note into the Notes field|The form should contain the entered note|
+|5|Press the Approve button|You should see "Travel Request was successfully updated"|
+
+#### Denying a travel request
+
+|Step|Action|Expected result|
```

Comment:
I think this test should follow the same steps las described[ in this comment](https://github.com/pulibrary/approvals/pull/1088/files#r1517788580) and not use the delegator option..